Development.md

```diff
@@ -0,0 +1,273 @@
+# Convox Rack Development Guide
```

Comment:
yes, that is at the beginning of Readme.md. This document is for engineers that already understand what Rack is and want to help develop.